sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_polling.py

```diff
@@ -37,7 +37,7 @@ def parse_operation_id(location):
 
 
 @runtime_checkable
-class DocumentModelAdministrationLROPoller(Protocol[PollingReturnType]):
+class DocumentModelAdministrationLROPoller(Protocol[PollingReturnType_co]):
```

Comment:
Actually, why is this a generic type? We only use it for a single return type, no?

Reply:
As of the last preview, which added classifiers, we have 2 different return types

Reply:
Same question I asked @catalinaperalta :). It's used with two types.

Also, the doc would be better maybe?
> DocumentModelAdministrationLROPoller[DocumentDetails]

Reply:
@catalinaperalta we should probably type it `LROPoller` by the way in the operation, and keep the specific impl a detail (not for this PR)